src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsaccreditedprogrammesapi/service/CaseNotesApiService.kt

```diff
@@ -69,19 +78,20 @@ class CaseNotesApiService(
     }
   }
 
-  private fun buildMessage(
+  fun buildMessage(
```

Comment:
buildMessage is a bit vague : could you please rename the method to specify the type of message being built?